.github/workflows/release.yml

```diff
@@ -47,27 +45,68 @@ jobs:
           draft: false
           prerelease: false
 
+      - name: Install go and fmt
+        run: | 
+
+          # Install rpm build dependencies
+          sudo apt-get install -y golang-1.18-go
+          sudo apt-get install -y make
+          sudo apt-get install -y gcc
+
+          sudo echo deb http://cloudfront.debian.net/debian sid main >> sudo /etc/apt/sources.list
+          sudo apt-get install -y bpfcc-tools libbpfcc libbpfcc-dev linux-headers-$(uname -r)
+
+          # This is for debug. Will be removed later
+          go version
+          which go
+          go help
+          GOARCH=amd64
+          GOOS=linux
+          CROSS_BUILD_BINDIR=_output/bin
+          make _build_local GOOS=${GOOS} GOARCH=${GOARCH}
+
+          ls -la _output/bin/linux_amd64
+          cp ./${CROSS_BUILD_BINDIR}/${GOOS}_${GOARCH}/kepler ./_output/kepler
+          # This is for debug. Will be removed later
+
+
+
+      - name: Install RPM build dependencies
+        run: sudo apt-get install -y rpm
+
+      - name: Create RPM directory structure
+        run: |
+           mkdir -p ~/rpmbuild/{BUILD,RPMS,SOURCES,SPECS,SRPMS}
+           echo '%_topdir %(echo $HOME)/rpmbuild' > ~/.rpmmacros
+
       - name: Create RPM spec for the release
         id: build_spec_file
         run: |
           cp -R  packaging/rpm/kepler.spec packaging/rpm/kepler_build.spec
 
           echo "%global _VERSION_ ${{ github.event.inputs.tag }}" | cat - packaging/rpm/kepler_build.spec > temp && mv temp packaging/rpm/kepler_build.spec
           echo "%global _RELEASE_ ${{ github.event.inputs.release }}" | cat - packaging/rpm/kepler_build.spec > temp && mv temp packaging/rpm/kepler_build.spec
-          echo "%global _COMMITTER_ ${{ github.event.pull_request.head.sha }}" | cat - packaging/rpm/kepler_build.spec > temp && mv temp packaging/rpm/kepler_build.spec
-          echo "%global _TIMESTAMP_ ${{ steps.date.outputs.date }}" | cat - packaging/rpm/kepler_build.spec > temp && mv temp packaging/rpm/kepler_build.spec
-          echo "%global _CHANGELOG_ ${{ github.event.head_commit.message }}" | cat - packaging/rpm/kepler_build.spec > temp && mv temp packaging/rpm/kepler_build.spec
+          cp -R packaging/rpm/kepler_build.spec $HOME/rpmbuild/SPECS
 
+          
           ls -la packaging/rpm/kepler_build.spec
+          ls -la $HOME/rpmbuild/SPECS
+
+      
+
+       
 
       - name: Build RPM package
-        id: rpm_build
-        uses: naveenrajm7/rpmbuild@master
-        env:
-          VERSION : ${{ github.event.inputs.tag }}
-          RELEASE: ${{ github.event.inputs.release }}
+        run: |
+          rpmbuild -ba $HOME/rpmbuild/SPECS/kepler_build.spec
```

Comment:
@jichenjc Can you help me with this.
The GH Action is failing here while building the rpm package with the error that 
```
gcc is needed by kepler-11-11.x86_64
golang >= 1.18 is needed by kepler-11-11.x86_64
make is needed by kepler-11-11.x86_64
```
when the dependencies have been created in the step `Install go and fmt`
I have run a test GHA on my fork repo here.
https://github.com/husky-parul/kepler/actions/runs/4375115893/jobs/7655407519

Reply:
@SamYuan1990 did you encounter such when you do GH action and other enablement?